main.py

```diff
@@ -163,7 +163,7 @@ def episode_finished(r):
                 save_dir = os.path.dirname(args.save_agent)
                 if not os.path.isdir(save_dir):
                     try:
-                        os.mkdir(save_dir, 0o755)
```

Comment:
just wondering to know if we need to manually create a directory (as mentioned in readme)  to save the model or the code above will do it automatically.

Reply:
we don't need to manually create directory, it will recursively save all the related dirs, i will change the readme, thanks!

Reply:
thank you